requirements.txt

```diff
@@ -1,7 +1,7 @@
-bittensor==5.2.0
+bittensor>=5.2.1,<5.3.0
 transformers<=4.28.0 
 wandb==0.15.3
-datasets==2.12.0
+datasets>=2.14.0
```

Comment:
Is there a reason that we moved from a strict `==` version requirement to a loose one? Personally, I think it's a better practice to pin the version or limited version so that we are unaffected by sudden new releases of `dataset`. What do you think?